builds/runtimes/python-3.6.3

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+# Build Path: /app/.heroku/python/
+# Build Deps: libraries/sqlite
+
+OUT_PREFIX=$1
+
+echo "Building Python…"
+SOURCE_TARBALL='https://python.org/ftp/python/3.6.3/Python-3.6.3.tgz'
+curl -L $SOURCE_TARBALL | tar xz
+mv Python-3.6.3 src
```

Comment:
I know this was pre-existing, but perhaps it would be good to move the version string out to a `PYTHON_VERSION` variable, to reduce the duplication?